osm-fudge/amenity-example.py

```diff
@@ -0,0 +1,26 @@
+"""
+Extract all objects with an amenity tag from an osm file and list them
+with their name and position.
+"""
+import osmium as o
+import sys
+
+class AmenityListHandler(o.SimpleHandler):
+
+    def print_amenity(amenity, tags, lon, lat):
+        name = tags.get('name', '')
+        print("%f %f %-15s %s" % (lon, lat, tags['amenity'], name))
+
+    def node(self, n):
+        if 'amenity' in n.tags:
+            self.print_amenity(n.tags, n.location.lon, n.location.lat)
```

Comment:
I think we also want to check that the name is there, no? Otherwise we can get amenities with empty name.

Reply:
Yes, that is the reason I was seeing empty lines in the output. We can ignore them.